artwork.json

```diff
@@ -256,5 +256,17 @@
       "github": "pultzlucas"
     },
     "license": "MIT"
+  },
+  {
+    "date": "2021-12-23",
+    "image": "/images/pride_deno.gif",
+    "title": "Pride Deno",
+    "alt": "Deno, but with queer pride~!",
```

Comment:
This is not a very good alt text. Alt text should describe the image.

Reply:
Edited it, good now?